python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/shape_lib_gen.py

```diff
@@ -1213,8 +1213,8 @@ def aten〇linalg_vector_norm(self: List[int], ord: float = 2, dim: Optional[Lis
 def aten〇frobenius_norm〇dim(self: List[int], dim: List[int], keepdim: bool = False) -> List[int]:
     return upstream_shape_functions.sum_mean_dim(self, dim, keepdim, 0)
 
-def aten〇upsample_nearest2d〇vec(input: List[int], output_size: Optional[List[int]], scale_factors: Optional[List[float]]) -> List[int]:
-    return upstream_shape_functions.upsample_nearest2d(input, output_size, scale_factors)
```

Comment:
Can you upstream the fixes to this function? Here is an example PR: https://github.com/pytorch/pytorch/pull/76889.

Reply:
Sure @ramiro050 I'm working on it - meanwhile can we aim to get this PR merged in ?

Reply:
Sure! Do update this once your Pytorch PR lands